crates/gpui/src/platform/blade/shaders.wgsl

```diff
@@ -227,7 +362,12 @@ fn vs_quad(@builtin(vertex_index) vertex_id: u32, @builtin(instance_index) insta
 
     var out = QuadVarying();
     out.position = to_device_position(unit_vertex, quad.bounds);
-    out.background_color = hsla_to_rgba(quad.background);
+    if (quad.background.tag == 0u) {
+        out.background_solid = hsla_to_rgba(quad.background.solid);
+    } else if (quad.background.tag == 1u) {
+        out.background_color0 = linear_to_srgba(hsla_to_rgba(quad.background.colors[0].color));
+        out.background_color1 = linear_to_srgba(hsla_to_rgba(quad.background.colors[1].color));
+    }
```

Comment:
So of I'm reading this right the background is set to `linear sRGB` if it's solid but converted to `non-linear sRGB` for gradients?

Reply:
Here is a bit of difference like the Metal, the `hsla_to_rgba` it was returns `linear sRGB`.

I kept the behavior of this function unchanged to avoid modifying other parts of the code.

And to convert the color in `vertex` stage to reduce convert color again in every `fragment` for performance.